tests/unit/hive_metastore/test_workflows.py

```diff
@@ -57,16 +56,16 @@ def test_migrate_ctas_views(run_workflow):
 @pytest.mark.parametrize(
     "workflow",
     [
-        TableMigration,
-        MigrateHiveSerdeTablesInPlace,
-        MigrateExternalTablesCTAS,
-        ScanTablesInMounts,
-        MigrateTablesInMounts,
+        TableMigration.update_migration_status,
+        MigrateHiveSerdeTablesInPlace.refresh_migration_status,
```

Comment:
Do we need to update other workflows as well with the logic you implemented? What do you think?

Reply:
I think so by look of it - I have replaced all the `refresh_migration_status` with `update_migration_status`.